src/main/java/org/jabref/logic/net/HttpPostService.java

```diff
@@ -0,0 +1,64 @@
+package org.jabref.logic.net;
```

Comment:
With [UniRest](http://kong.github.io/unirest-java/#requests), this should be obsolete. See `Asynchronous Requests` in that site.

Reply:
We removed this class and used URLDownload.java instead